### ISSUE TITLE
Recommended input reference of scMidas

### ISSUE THREAD

**Issue by WhatMelonGua** [labomics/midas#13]
Dear author, hello!
At present, my application scenario is that I have a portion of multimodal data (RNA+ATAC), among which the transcriptome data has high quality, but only about 10% of cells with good ATAC quality. When integrating, should I keep input the low-quality ATAC matrix directly, or del the atac matrix?(I'm not sure low-quality ATAC inputs if will influence midas performance) I hope to receive your answer! thank you very much
Another point is, can I use the GeneScore output by ArchR for the input ATAC matrix? Or TileMatrix (which is usually binary)?

**Comment by zhen-he** [comment]
Hello,

I suggest removing the low-quality ATAC matrix because it does not meet quality control standards and could affect integration performance. After removal, you can split the data into two batches: one with RNA only, and another with RNA+ATAC. Additionally, it’s best to ensure that each batch contains more than 5,000 cells, especially for multi-omics data, to avoid overfitting.

As for GeneScore, its dimensions are the same as RNA, which current models are not fully compatible with. It is recommended to input a binarized peak matrix instead. If you wish to use GeneScore, you can modify the model to add a new modality for GeneScore. This modality should be similar to RNA, but with continuous values as input instead of counts, which may also require modifying the original BCE loss function.

**Comment by WhatMelonGua** [comment]
Thank you very much! Your answer was very detailed and perfectly resolved my confusion.
I will close this issue, and express my sincere gratitude once again.🌹🌹🌹
